templates/expo-template-default/package.json

```diff
@@ -19,7 +19,7 @@
     "@expo/vector-icons": "^14.0.2",
     "@react-navigation/bottom-tabs": "^7.2.0",
     "@react-navigation/native": "^7.0.14",
-    "expo": "~52.0.11",
+    "expo": "canary",
```

Comment:
not necessary to change these, they will be updated automatically when we publish the templates with the canary script. these should point to whatever the current version is inside of the expo repo while on main